src/components/organisms/Dashboard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Pane, Select, Text } from 'evergreen-ui';
+import AddComponentButton from '../atoms/AddComponentButton';
+import DashboardItem from '../molecules/DashboardItem';
+import { useReactiveVar } from '@apollo/client';
+import { dashboardItemsVar } from '../../cache';
+import DataWrapper from '../molecules/DataWrapper';
+import ThresholdChart, { TimeEntry } from '../atoms/ThresholdChart';
+import { ParentSize } from '@visx/responsive';
+import { MetApiCompactAirTemperature } from '../../types/compact';
+import { formatMetData } from '../../utils/formatMetData';
+
+const Dashboard: React.FC = () => {
+    //Apollo local state
+    const dashboardItems = useReactiveVar(dashboardItemsVar);
+
+    return (
+        <Pane width="100%" height="100%" display="flex" flexDirection="column" alignItems="center" paddingBottom="20px">
+            <Pane width="80%" height="100px" position="relative">
+                <Select position="absolute" bottom="20px" width="200px">
```

Comment:
Why do these two use relative/absolute positioning? 

Is it possible to position them using grid/flex instead?

Reply:
should be possible 👍

Reply:
just wanted to see what it would look like